docs/_source/getting_started/installation/configurations/user_management.md

```diff
@@ -11,15 +11,15 @@ Let's first describe Argilla's user management model:
 A Argilla user is defined by the following fields:
 
 - `username`: The username to use for login into the Webapp.
-- `email`(optional): The user's email.
-- `fullname` (optional): The user's full name
-- `disabled`(optional): Whether this use is enabled (and can interact with Argilla), this might be useful for disabling user access temporarily.
-- `workspaces`(optional): The team workspaces where the user has read and write access (both from the Webapp and the Python client). If this field is not defined the user will be a super-user and have access to all datasets in the instance. If this field is set to an empty list `[]` the user will only have access to her user workspace. Read more about workspaces and users below.
+- `first_name`: The user's first name
+- `last_name` (optional): The user's last name
+- `role`: The user's role in Argilla. Available roles are: "admin" and "annotator". Only "admin" users can create and delete workspaces and datasets
+- `workspaces` : The workspaces where the user has read and write access (both from the Webapp and the Python client). If this field is not defined the user will be a super-user and have access to all datasets in the instance. If this field is set to an empty list `[]` the user will only have access to her user workspace. Read more about workspaces and users below.
 - `api_key`: The API key to interact with Argilla API, mainly through the Python client but also via HTTP for advanced users.
 
 ### `Workspace`
 
-A workspace is a Argilla "space" where users can collaborate, both using the Webapp and the Python client. There are two types of workspace:
+A workspace is a Argilla "space" where users can collaborate, both using the Webapp and the Python client:
 
 - `Team workspace`: Where one or several users have read/write access.
 - `User workspace`: Every user gets its own user workspace. This workspace is the default workspace when users log and load data with the Python client. The name of this workspace corresponds to the username.
```

Comment:
The concept of "user workspace" has no more sense. Workspaces can be created dynamically, and users can be assigned to workspaces using the API. Admin users can work on all workspaces, and annotators cannot create datasets. 

```suggestion
```